include/glow/Runtime/Provisioner/Provisioner.h

```diff
@@ -51,6 +54,13 @@ class Provisioner final {
   /// Padding factor to account for generated code size. Should be greater
   /// than 1.0.
   const float NETWORK_PADDING_FACTOR = 1.1;
+
+  /// Vector of DeviceManagers.
+  std::vector<std::shared_ptr<DeviceManager>> devices_;
+
+  /// Index of current deviceManager. This allows a round robin loading of
+  /// deviceManagers.
+  unsigned int currDevice_{0};
```

Comment:
nit: current is a weird idea since state of the provisioner is a bit abstract, maybe `nextDevice_` ?